app/src/views/GrantRegistryNewGrant.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <div class="flex flex-col justify-center sm:px-6 lg:px-8">
+    <div class="sm:mx-auto sm:w-full sm:max-w-md">
+      <img
+        class="mx-auto h-12 w-auto"
+        src="https://tailwindui.com/img/logos/workflow-mark-indigo-600.svg"
+        alt="Workflow"
+      />
+      <h2 class="mt-6 text-center text-3xl font-extrabold text-gray-900">Create New Grant</h2>
+    </div>
+
+    <div class="mt-8 sm:mx-auto sm:w-full sm:max-w-md text-left">
+      <div class="py-8 px-4 border shadow sm:rounded-lg sm:px-6">
+        <form class="space-y-6" @submit.prevent="createGrant">
+          <!-- Owner address -->
+          <BaseInput
+            v-model="form.owner"
+            description="The owner has permission to edit the grant"
+            id="owner-address"
```

Comment:
When in dark mode its difficult to see the data in the fields.

Reply:
Good catch, I forgot to check how dark mode looked. Made some improvements in 6f68b3f0ba8ce6c631d811373389117b1ab8d485 so it should look better now